firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidator.java

```diff
@@ -35,13 +35,13 @@ final class FirebasePerfTraceValidator extends PerfMetricValidator {
   }
 
   /**
-   * Validates a trace metric, it validates root trace and subtraces and checks if the name, deep
-   * and durations are valid.
+   * Validates a trace metric, it validates root trace and checks if the name, deep and durations
```

Comment:
nit: what do you mean by deep? "depth"?

Reply:
I think `deep` is talking about `isValidTrace(TraceMetric trace, int deep)`. However since we are removing `deep` this comment should also get rid of "deep".